libs/core/src/lib/product-switch/product-switch/product-switch.component.html

```diff
@@ -8,6 +8,7 @@
         [disabled]="disabled"
         [triggers]="triggers"
         [focusTrapped]="false"
+        [noArrow]="noArrow"
```

Comment:
it needs to take a property in the .ts file that is noArrow

Reply:
This should be [noArrow]="false"

Reply:
@mikerodonnell89 don't we want to keep it as a parameter, instead of always hardcoding it?